docs/modules/ROOT/partials/install/prepare-installer.adoc

```diff
@@ -74,6 +82,16 @@ openshift-install --dir "${INSTALLER_DIR}" \
 yq w -i "${INSTALLER_DIR}/manifests/cluster-ingress-02-config.yml" \
   spec.domain apps.example.com
 ----
++
+. Cilium Optional: Copy pre-rendered Cilium manifests
++
+[%collapsible]
+====
+[source,bash]
+----
+cp catalog/manifests/cilium/olm/* target/manifests/
```

Comment:
This will show up in the Exoscale install instructions, but the catalog won't be present, because the early catalog rendering is only part of the cloudscale.ch install instructions.